templates.md

```diff
@@ -138,6 +138,16 @@ To overwrite a section entirely, you may use the `overwrite` statement:
 
 	{{-- This comment will not be in the rendered HTML --}}
 
+#### Service injection
+We can use `@inject` to resolve services from the service container.
+
+	@inject('html', 'HtmlBuilder')
```

Comment:
Would you rather use an example that is in the Laravel framework (if so, which would be appropriate?)? I think the HtmlBuilder is a nice example as it's probably better to only inject services that really matter when working with html. (vs for example a `StatsService` that could also be included using the `with` function.